Anchor.toml

```diff
@@ -15,4 +15,8 @@ cluster = "Localnet"
 wallet = "keys/localnet/admin-bossj3JvwiNK7pvjr149DqdtJxf2gdygbcmEPTkb2F1.json"
 
 [scripts]
-test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
\ No newline at end of file
+test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
+
+[[test.genesis]]
```

Comment:
Could you remove this, we dont use transfer hook anymore

Reply:
Yeah, but I still want to create a test with transfer hook supported tokens!